invokeai/app/services/invocation_services.py

```diff
@@ -19,6 +20,7 @@ class InvocationServices:
     queue: InvocationQueueABC
     model_manager: ModelManager
     restoration: RestorationServices
+    configuration: InvokeAISettings=None
```

Comment:
Why does the config service have a default value here?

Reply:
This looks like an error and has been removed.